src/containers/BirdsBlock/BirdsBlock.tsx

```diff
@@ -0,0 +1,101 @@
+import styles from './BirdsBlock.module.scss';
+
+import { Bird } from '@containers/Bird/Bird';
+import { DescriptionBirdDefault } from '@view/DescriptionBirdDefault/DescriptionBirdDefault';
+import { MouseEventHandler, useState } from 'react';
+import { ButtonNavigation } from '@view/ButtonNavigation/ButtonNavigation';
+import { DescriptionBird } from '@view/DescriptionBird/DescriptionBird';
+
+import { ADD_POINTS } from '@constans/reducerActionType';
+import { useDispatch } from 'react-redux';
+import { IBird } from '@constans/interface/birds.interface';
+import { IPropsValueBirds } from './BirdsBlock.interface';
+
+import useSound from 'use-sound';
+import alarmRight from '@static/alarmValue/right.mp3';
+import alarmError from '@static/alarmValue/error.mp3';
+
+const BirdsBlock: React.FC<IPropsValueBirds> = ({ birds, url, setMainBird, setPercentage, percentage }) => {
+  const [bird, addBird] = useState<IBird[]>([]);
+  const [disabled, setDisabled] = useState(true);
+  const [styleBtn, setStyleBtn] = useState(`${styles.nexBtn}`);
+  const [points, setPoints] = useState(5);
+  const [audioControl, setAudioControl] = useState(0);
+
+  const [birdsArr, setBirdsArr] = useState<IBird[]>([]);
+
+  const dispatch = useDispatch();
+
+  const [playError] = useSound(alarmError);
+  const [playRight] = useSound(alarmRight);
+
+  const addPoints = (points: number) => {
+    dispatch({ type: ADD_POINTS, payload: points });
+  };
+
+  const onBirdNameClick = (bird: IBird) => {
+    setAudioControl(0);
+
+    if (bird) {
+      addBird([bird]);
+    }
+    if (!bird?.isRightAnswer) {
+      points ? setPoints(points - 1) : setPoints(0);
+      if (disabled) playError();
+    }
+    if (bird?.isRightAnswer) {
+      const [mainBird] = birds.filter((bird: IBird) => bird.isRightAnswer);
+      setMainBird(mainBird);
+      setDisabled(false);
+      setStyleBtn('nextBtnRight');
+      addPoints(points);
+      if (disabled) playRight();
+    }
+    if (disabled) {
+      setBirdsArr([...birdsArr, bird]);
+    }
+  };
+
+  const clearList = () => {
+    addBird([]);
+    setMainBird({
+      name: '******',
+      img: 'https://birds-quiz.netlify.app/static/media/bird.06a46938.jpg',
+    });
+
+    setStyleBtn('');
+    setDisabled(true);
+    setPoints(5);
+    setPercentage(0);
+    setBirdsArr([]);
+  };
+
+  return (
+    <div>
+      <div className={styles.ValueBlockBirds}>
+        <ul className={styles.valuesMenu}>
+          {birds.map((bird: IBird) => {
+            const stateStyleValue = birdsArr?.find((el: IBird) => el.id === bird.id);
+            return (
+              <Bird
+                key={bird.id}
+                onBirdNameClick={onBirdNameClick}
+                bird={bird}
+                designType={bird.isRightAnswer ? 'right' : 'error'}
```

Comment:
Бессмысленно что ты передаешь дизайн тип исходя из **статических данных**, которые в массиве. Ты тут должен передавать designType исходя из birdsArr.

const birdDesignType = foundBird?.isRightAnswer ? 'right' : 'error'

```suggestion
                designType={foundBird ? getBirdDesignType(bird) : undefined}
```

Иначе с этим stateStyleValue получается каша полная. Если я тебе завтра скажу добавить дополнительный дизайн стиль с желтым кружочком, то у тебя будут большие проблемы, хотя это одно маленькое изменение.